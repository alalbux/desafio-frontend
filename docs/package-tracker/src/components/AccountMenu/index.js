'use client';
import React, { useState } from 'react';
import Navigation from '@/components/Navigation';
import LoggedInContent from '@/components/LoggedInContent';
import LoggedOutContent from '@/components/LoggedOutContent';

const AccountMenu = () => {
  const [isLogged, setIsLogged] = useState(false);

  return (
    <div>
      <Navigation data-testid="navigation" />
      {isLogged ? (
        <LoggedInContent
          data-testid="logged-in-content"
          setIsLogged={setIsLogged}
        />
      ) : (
        <LoggedOutContent
          data-testid="logged-out-content"
          setIsLogged={setIsLogged}
        />
      )}
    </div>
  );
};

export default AccountMenu;
