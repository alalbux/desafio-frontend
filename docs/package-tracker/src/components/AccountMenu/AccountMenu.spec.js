import React from 'react';
import { render } from '@testing-library/react';
import AccountMenu from '@/components/AccountMenu';

describe('AccountMenu', () => {
  it('matches the snapshot', () => {
    const { container } = render(<AccountMenu />);
    expect(container).toMatchSnapshot();
  });
});
