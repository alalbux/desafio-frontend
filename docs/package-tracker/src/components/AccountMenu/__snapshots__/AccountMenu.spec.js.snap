// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountMenu matches the snapshot 1`] = `
<div>
  <div>
    <nav>
      <div
        style="display: flex;"
      >
        <div
          style="position: relative; margin: 0px 1.5rem;"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Para vocÃª
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          style="position: relative; margin: 0px 1.5rem;"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Para empresas
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          style="position: relative; margin: 0px 1.5rem;"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Para entregar
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          style="position: relative; margin: 0px 1.5rem;"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Venda e suporte
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          style="position: relative; margin: 0px 1.5rem;"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Acompanha seu pedido
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </nav>
    <div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-button MuiButton-buttonText.primary MuiButton-sizeMedium MuiButton-buttonSizeMedium MuiButton-root MuiButton-button MuiButton-buttonText.primary MuiButton-sizeMedium MuiButton-buttonSizeMedium css-1w1rijm-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        Entrar
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        Criar conta
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;
